nemo_text_processing/text_normalization/it/taggers/tokenize_and_classify.py

```diff
@@ -109,28 +111,38 @@ def __init__(
             )
 
             punct = pynutil.insert("tokens { ") + pynutil.add_weight(punct_graph, weight=1.1) + pynutil.insert(" }")
-            punct = pynini.closure(
-                pynini.compose(pynini.closure(NEMO_WHITE_SPACE, 1), delete_extra_space)
-                | (pynutil.insert(" ") + punct),
-                1,
-            )
+
```

Comment:
Ditto, I don't like commented out code in production databases. This should only be here if developers need the option to switch between. In that case, it should be clear why the commented out code was provided.